includes/class-notices.php

```diff
@@ -209,6 +209,14 @@ public function render_admin_notices() {
 			return;
 		}
 
+		// Do not render notice on Gutenberg editor page.
```

Comment:
@Sidsector9 @DevinWalker For now we can use the Gutenberg core function to detect editor page.